definitions/json/CaseField/CaseField-Multiples-nonprod.json

```diff
@@ -40,6 +40,20 @@
     "Label": "Case File View",
     "FieldType": "Label",
     "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "ID": "applicationsTab",
+    "Label": "### Applications",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales_Multiple",
+    "ID": "applicationsLinks",
+    "Label": "<a href=\"/cases/case-details/${[CASE_REFERENCE]}/\">All applications</a> <br> <a href=\"/cases/case-details/${[CASE_REFERENCE]}/\">View an application</a>",
```

Comment:
These links arent correct, they should take you to the event (which doesn't exist atm but will use the same name as its single counterpart)

Reply:
Purposely didn't include broken links but I've added what we expect them to be now.